recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,19 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,16 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if not "PYSDL2_DLL_PATH" in os.environ:
```

Comment:
Shouldn't this be `"PYSDL2_DLL_PATH" not in os.environ`?  (not "PYSD...") might evaluate to false depending on the precede and parentheses.  I think using "not in" is more readable and cleaner.